content/html/encyclopedia/tables.md

```diff
@@ -0,0 +1,108 @@
+---
+Subjects:
+  - "HTML/CSS"
+  - "Web Development"
+Tags: 
+  - "Elements"
+  - "HTML & CSS"
+  - "Table"
+  - "Web Development"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+# HTML - Tables
+
+In HTML, a table is an element that allows for the representation of data in two dimensions; these dimensions are columns and rows.
+
+## Structure
+
+Tables are composed of multiple smaller types of elements that have special meaning within the structure of the table.  The elements that are used in the construction of a typical table are as follows:
+
+### Table
+
+The `<table>` element itself serves as the container for all elements and information contained within the table including headers, columns, and rows.
+
+### Table Row
+
+The table row element, denoted as `<tr>`, is used to display data within a row (horizontally) in a table. Table rows do not display data on their own. Instead, table rows serve as a container for child elements that are responsible for displaying the data in a semantic way.
+
+```html
+<table>
+  <tr></tr> <!-- Row 1 -->
+  <tr></tr> <!-- Row 2 -->
+  <tr></tr> <!-- Row 3 -->
+</table>
+```
+
+### Table Header
+
+The table header element, denoted as `<th>`, is used to display data for a specific column header within a table. Multiple `<th>` elements can be added as children to a table row at the top of a table to create a complete header for a table.
+
+```html
+<table>
+  <tr>
+    <th>Column 1</th>
+    <th>Column 2</th>
+    <th>Column 3</th>
+  </tr>
+</table>
+```
+<code>
```

Comment:
for this one, we might have to do a table in markdown!

https://www.markdownguide.org/extended-syntax/